dev/loadgen/pkg/loadgen/executor.go

```diff
@@ -155,6 +162,9 @@ func (w *WsmanExecutor) Observe() (<-chan WorkspaceUpdate, error) {
 
 // StopAll stops all workspaces started by the executor
 func (w *WsmanExecutor) StopAll() error {
+	for _, s := range w.Sub {
+		s()
+	}
 	fmt.Println("kubectl delete pod -l component=workspace")
```

Comment:
This line looks outdated. This will delete all the ws pods including the one not generated by this load test. Can we use other label like `owner=0000....` and possibly introduce additional label like `loadgen=true`?

Reply:
This line does not delete workspaces, but rather cancels all subscriptions previously created using the `Observe` calls. `w.Sub` is a list of `context.CancelFunc` which cancel the subscription contexts.

`loadgen` currently does not delete any workspace to allow for inspection of the state after all workspaces have been started.